contracts/LifToken.sol

```diff
@@ -455,47 +456,64 @@ contract LifToken is Ownable, ERC20, SafeMath, PullPayment {
 
     }
 
-    // ERC20 transfer method but with data parameter.
+    //ERC20 token approve method with data call/log option.
+    function approveData(address spender, uint value, bytes data, bool doCall) onStatus(3,4) returns (bool success) {
```

Comment:
Oh right, so the spender is the one who spends the token from some other token holder. So now the name makes more sense ;) :+1: